index.html

```diff
@@ -185,6 +199,19 @@
         }
 
         tick () {
+          counter += 1
+          if (counter % 15 == 0) {
+            let stats = [
```

Comment:
We should really break up your save state into something other than an array. I know I've said this before. But lets say you want to save any other data ie: `userAvatarName`. It'd be impossible to add to an array. Also you don't have to do something complicated like you did below. 

kudos on the neat concat solution tho :)